definitions/variable/technology/electricity-operation.yaml

```
@@ -91,3 +92,19 @@
     description: Marginal Cost associated to the Frequency Containment requirement
     unit: EUR_2020/MWh
     skip-region-aggregation: true
+    
+- Export|Electricity:
+    description: Exported electricity from a given region
+    unit: MWh
+- Import|Electricity:
+    description: Exported electricity from a given region
+    unit: MWh
+- Non served Demand|Electricity:
+    description: Not served demand
+    unit: MWh
+- Charged Energy|Electricity|{Electricity Input}:
+    description: Amount of energy charged to the storage during the period
```

If you think having two separate variables is the right approach, I suggest to add the following note to avoid confusion between injection (charging) and extraction (discharging).

```suggestion
    description: Amount of energy charged to the storage during the period
    notes: This has to be a positive value. If more energy is discharged from storage than charged into storage, please used the variable "Discharged Energy"
```

---

@danielhuppmann @erikfilias the idea here is that the 2 variables can be positive at the same time, eg if you have 3 time steps in the period covered by the variable and energy is discharged in the 1st time step and charged in the other timesteps, then the Charged energy will be the sum of the 2 last timesteps while Discharged Energy will be the 1st timestep. I add a comment to explain that in the variable. 
@erikfilias agree?

---

I also agree. In [openTEPES](https://opentepes.readthedocs.io/en/latest/MathematicalFormulation.html), we use the same concept of having two variables.